distr_tracing/distr_tracing_otel/distr-tracing-otel-installing.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="install-distributed-tracing-otel"]
+= Installing the {OTELShortName}
```

Comment:
"Installing the distributed tracing data collection" reads a bit odd to me without "Operator", maybe use the full form for this?

Reply:
I did mention it to the team that the word choice of "collection" was not the most fortunate one.

BTW, we already have a Jira opened by my DPM to revisit and possibly rethink the structure of this page (in terms of how many modules is optimal) and same for the other installation pages after this PR gets merged.